packages/react/src/blocks/Text.tsx

```diff
@@ -55,7 +55,7 @@ class TextComponent extends React.Component<TextProps> {
           return (
             <React.Fragment>
               {/* TODO: <BuilderEditableText component that wraps this for other components with text */}
-              <span
+              <div
```

Comment:
what do you think of limiting this PR changes to the gen2 sdks only?

Reply:
I think that's a good idea, we can revisit the idea in gen1 before the next major release.